atlite/datasets/era5.py

```diff
@@ -44,7 +44,7 @@ def nullcontext():
 
 features = {
     "height": ["height"],
-    "wind": ["wnd100m", "wnd_azimuth", "roughness"],
+    "wind": ["wnd100m", "wnd_shear_exp", "wnd_azimuth", "roughness"],
```

Comment:
Any problems if the new extrapolation functions are used with an old cutout?

Reply:
You would raise a `KeyError` when `atlite.wind.extrapolate_wind_speed` tries to access `ds["wnd_shear_exp"]`.

We could raise a more informative error message instead, i guess.

Reply:
Done now